codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/DafnyToAwsSdkShapeVisitor.java

```diff
@@ -50,6 +48,7 @@ public class DafnyToAwsSdkShapeVisitor extends ShapeVisitor.Default<String> {
   private final ServiceTrait serviceTrait;
   private final boolean isOptional;
   private final boolean isPointable;
+  public static final Map<MemberShape, String> visitorFuncMap = new HashMap<>();
```

Comment:
SNAKE_CASE.

Reply:
This was updated to be a private variable. Not necessarily for private var